assets/airports/mdsd.json

```diff
@@ -0,0 +1,4005 @@
+        "KARIB": {
+            "icao": "KARIB",
+            "name": "Kareeb",
+            "suffix": {"17":"1G", "35":"1H"},
+            "entryPoints": {
+                "POKAK": ["POKAK"]
+            },
+            "body": [["KARIB", "A190|S250-"]],
+            "rwy": {
+                "17": [["SD116", "A90|S220-"], ["VIPKI", "A70|S210-"], ["SD119", "A50|S210-"], ["EMULU", "A30|S210-"]],
+                "35": [["SD124", "A80|S220-"], ["OTVOG", "A30|S210-"]]
+            },
+            "draw": [["KARIB", "SD116", "VIPKI"], ["KARIB", "SD124", "OTVOG"]]
+        },
+        "GOSUL": {
+            "icao": "GOSUL",
+            "name": "Gohsool",
+            "suffix": {"17":"1G", "35":"1H"},
+            "entryPoints": {
+                "ANTEX": ["ANTEX"]
+            },
+            "body": [["GOSUL", "A190|S250-"]],
+            "rwy": {
+                "17": [["SD114", "A120|S220-"], ["SD115", "A100|S220-"], ["BOKLO", "A60|S210-"], ["SD119", "A50|S210-"], ["EMULU", "A30|S210-"]],
+                "35": [["SD123", "A50|S220-"], ["OTVOG", "A30|S210-"]]
+            },
+            "draw": [["GOSUL", "SD114", "SD115", "BOKLO", "SD119"], ["GOSUL", "SD123", "OTVOG"]]
+        },
+        "KODIX": {
+            "icao": "KODIX",
+            "name": "Kohdeex",
+            "suffix": {"17":"1G", "35":"1H"},
+            "entryPoints": {
+                "ONPAD": ["ONPAD", ["POBIN", "A290-|S300"], ["BIBAS", "A190|S270"]]
+            },
+            "body": [["KODIX", "A140|S250-"]],
+            "rwy": {
+                "17": [["SD111", "A70|S220-"], ["KEPTI", "A30|S210-"]],
+                "35": [["SD120", "A70|S220-"], ["LETIR", "A50|S210-"], ["OTVOG", "A30|S210-"]]
+            },
+            "draw": [["KODIX", "SD111", "KEPTI"], ["KODIX", "SD120", "LETIR"]]
+        },
+        "OKOSO": {
+            "icao": "OKOSO",
+            "name": "Ohkohsoh",
+            "suffix": {"17":"1G", "35":"1H"},
+            "entryPoints": {
+                "ETBOD": ["ETBOD"]
+            },
+            "body": [["OKOSO", "A160|S250-"]],
+            "rwy": {
+                "17": [["SD110", "A80|S220-"], ["KEPTI", "A30|S210-"]],
+                "35": [["SD127", "A100|S220-"], ["URLOR", "A40|S210-"], ["VINRO", "A30|S210-"]]
+            },
+            "draw": [["OKOSO", "SD110", "KEPTI"], ["OKOSO", "SD127", "URLOR", "VINRO"]]
+        },
+        "RIBES": {
+            "icao": "RIBES",
+            "name": "Reebaes",
+            "suffix": {"17":"1G", "35":"1H"},
+            "entryPoints": {
+                "PTA": ["PTA", ["BORAS", "A190|S270"]]
+            },
+            "body": [["RIBES", "A140|S250-"]],
+            "rwy": {
+                "17": [["SD112", "A70|S220-"], ["KEPTI", "A30|S210-"]],
+                "35": [["SD121", "A70|S220-"], ["LETIR", "A50|S210-"], ["OTVOG", "A30|S210-"]]
+            },
+            "draw": [["RIBES", "SD112", "KEPTI"], ["RIBES", "SD121", "LETIR"]]
+        }
+    },
+    "spawnPatterns": [
```

Comment:
From the [DR Air Transport Statistics Report (2016)](http://www.jac.gob.do/transparencia/images/docs/estadisticas/Air%20Transport%20Statistics%20Report%20-%20Dominican%20Republic%202016.pdf), the 2016 stats indicate 33,555 aircraft movements. IAW the [traffic level specs](https://github.com/openscope/openscope/blob/develop/documentation/airport-file-standards.md), this should fall between 3.8 and 11.5acph. Currently, there's a total of 29acph.

Please reduce the traffic level.